Dockerfile

```diff
@@ -13,6 +13,6 @@ RUN useradd --create-home --shell /bin/bash spfx && \
 
 USER spfx
 
-RUN npm i --location=global gulp@4 yo @microsoft/generator-sharepoint@1.15.2
+RUN npm i --location=global gulp@4 yo @microsoft/generator-sharepoint@1.16.0
```

Comment:
Let's align with the latest guidance and install `gulp-cli` instead of `gulp` which is the correct way forward

Reply:
should we install gulp-cli 2.3.0 then?

Reply:
I don't think there's a version restriction but to err on the safe side, let's restrict it to @2, rather than a specific full version

Reply:
For repeatability it might be a good idea to tie it to a specific version but let's start with @2 for now and then we see.